doc/lua_api.txt

```diff
@@ -6201,6 +6198,21 @@ Used by `minetest.hud_add`. Returned by `minetest.hud_get`.
         size = { x=100, y=100 },
     --  ^ Size of element in pixels
     }
+    
+Player HUD Flags
```

Comment:
Not sure about putting this in definition tables, as it seems out of place, how about in the earlier HUD section?